src/System.Windows.Forms/src/System/Windows/Forms/Control.cs

```diff
@@ -10743,12 +10725,8 @@ public void SetBounds(int x, int y, int width, int height, BoundsSpecified speci
         [EditorBrowsable(EditorBrowsableState.Advanced)]
         protected virtual void SetBoundsCore(int x, int y, int width, int height, BoundsSpecified specified)
         {
-#if DEBUG
-            if (CompModSwitches.SetBounds.TraceInfo)
-            {
-                Debug.WriteLine(string.Format(CultureInfo.CurrentCulture, "{0}::SetBoundsCore(x={1} y={2} width={3} height={4} specified={5}", Name, x, y, width, height, specified));
-            }
-#endif
+            Debug.WriteLineIf(CompModSwitches.SetBounds.TraceInfo,
+                $"{Name}::SetBoundsCore(x={x} y={y} width={width} height={height} specified={specified}");
```

Comment:
Could you add a missing closing parenthesis here?
```suggestion
                $"{Name}::SetBoundsCore(x={x} y={y} width={width} height={height} specified={specified})");
```